tests/containers/rootless_podman.pm

```diff
@@ -36,6 +36,7 @@ sub run {
 
     my $podman = $self->containers_factory('podman');
 
+    $podman_version = get_podman_version();
```

Comment:
this should really be above line #155 inside `verify_userid_on_container`, it's not being used anywhere else in this method and can cause issues down the line

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18733